README.md

```diff
@@ -31,7 +31,7 @@ Once the dependencies are installed, you can build AIKIDO using [CMake]:
 ```shell
 $ mkdir build
 $ cd build
-$ cmake ..
+$ cmake -DCMAKE_PREFIX_PATH=/opt/ros/indigo ..
```

Comment:
> Is Aikido just always dependent on ROS now? Why is this not being split into a new section like "Compiling for a ROS workspace"?

No, it should still build without ROS. All of the ROS-dependent components are optional.